_drafts/SC4-1-1-unique-id.md

```diff
@@ -0,0 +1,86 @@
+---
+name: unique element id attributes
+description: |
+  This rule checks that all `id` attribute values on a single page are unique.
+
+success_criterion:
+- 4.1.1 # Success Criterion 4.1.1 (Parsing)
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Bryn Anderson
+- Jey Nandakumar
+- Kamyar Rasta
+- Wilco Fiers
+---
+
+## Test procedure
+
+### Applicability
+
+Any element with an `id` attribute.
+
+### Expectation
+
+All `id` attribute values are unique.
```

Comment:
I'd rephrase it along the lines of:

> Each test target has an `id` attribute that is unique within the [document context](#insert-link-here) of the element.

Reply:
The term "document context" might not be the best fit; what I want to express is the union of browsing contexts and shadow hosts.

Reply:
I have changed this now inline with your first comment.